holoviews/tests/core/test_dynamic.py

```diff
@@ -78,11 +78,6 @@ def test_simple_constructor_streams_dict_parameter(self):
     def test_simple_constructor_streams_dict_class_parameter(self):
         DynamicMap(lambda x: x, streams=dict(x=ExampleParameterized.param.example))
 
-    def test_simple_constructor_streams_dict_invalid(self):
-        regexp = "Cannot handle value 3 in streams dictionary"
-        with self.assertRaisesRegex(TypeError, regexp):
-            DynamicMap(lambda x: x, streams=dict(x=3))
```

Comment:
Why does this not work anymore?

Reply:
I'm allowing constants in the refs dictionary. I think it's convenient in the same way a partial is convenient but I guess I'd be okay restoring an error if all values are constant.

Reply:
I'm unsure whether the extra convenience is worth the potential foot gun.  

I will trust your judgment, so please do what you think is best.

Reply:
Agreed, passing the parameter value rather than a parameter object is a frequent source of confusion so we shouldn't make that more likely.

Reply:
Fixed.